Headers/DebugServer2/Host/Linux/ExtraWrappers.h

```diff
@@ -31,6 +31,8 @@
 #endif // ARCH_ARM && !ARM_VFPREGS_SIZE
 
 #if defined(ARCH_X86) || defined(ARCH_X86_64)
+const size_t x87Padding = 6;
```

Comment:
You should define this where you use it. It's not something we want to export to other parts of the code.
Also, you can make it `static constexpr` instead of `const`.

Reply:
It will be defined in a few places then, which I don't find nicer than this at all. However, `static constexpr` does sound nicer :D

Reply:
Ah you're right. I didn't see how much it was used.